include/GafferScene/Light.h

```diff
@@ -55,6 +55,9 @@ class GAFFERSCENE_API Light : public ObjectSource
 		Gaffer::Plug *parametersPlug();
 		const Gaffer::Plug *parametersPlug() const;
 
+		Gaffer::BoolPlug *includeInDefaultLightSetPlug();
```

Comment:
The verbosity of this is bugging me a bit, but I'm not sure what would be better :

- `illuminateByDefault` : shorter but doesn't mention the set, which is useful to know about
- `defaultLightSet` : shorter, mentions the set, but sounds odd
- `defaultLight` : shortest, _almost_ mentions the set, sounds OK

Any thoughts?

Reply:
Sounds good.